packages/core-web/src/styles/markbind.css

```diff
@@ -175,6 +175,38 @@ footer {
     text-decoration: underline;
 }
 
+.red {
+    color: red;
```

Comment:
this is a good case for using `!important`.

we can make the class name something more specific / conflict free too, e.g. follow bootstrap's `text-xx` but prefix something more. 👀

Reply:
Made these tiny changes and merged. Nice work here! @weiquu